README.md

```diff
@@ -53,16 +53,32 @@ This can be done by configuring an IDE or executing the following command in a t
 
 > If you intend to use the framework multithreading option, be aware of the need to synchronize the access to output files or state manipulated by the implementations of the framework variability points.
 
-For example, for running the study we use as an example to illustrate the variability points at the beginning of the page, we invoke the following command at the project top folder: 
-* Windows/Linux/Mac: `groovy -cp src src/main/app/MiningFramework.groovy --access-key github-personal-access-token --threads 2 ./projects.csv SOOTAnalysisOutput`
-
-> For the used variability point implementation, the provided GitHub [personal access token](https://github.com/settings/tokens) (opt for repo scope) should be associated with a GitHub account also registered in [Travis](https://travis-ci.org/). Forks will be created for each project, the builds will be generated via Travis, and deployed to the forks as GitHub releases.
+> The CLI has the following help page:
+```
+usage: miningframework [options] [input] [output]
+the Mining Framework take an input csv file and a name for the output dir
+(default: output)
+ Options:
+ -a,--access-key <access key>   Specify the access key of the git account
```

Comment:
git account used for what? 

deixar claro que essa opcao é usada apenas por implementacoes de interface que precisam de acesso ao GitHub...